utils/build_swift/tests/expected_options.py

```diff
@@ -202,22 +205,52 @@ def sanitized_str(self):
         raise ValueError('invalid option_string format: ' + self.option_string)
 
 
-class Option(_BaseOption):
-    """Option that accepts no arguments."""
+class HelpOption(_BaseOption):
+    """Option that prints the help message and exits."""
+
+    pass
+
+
+class SetOption(_BaseOption):
+    """Option that accepts no arguments, setting the destination to a
+    hard-coded value or None.
+    """
 
     def __init__(self, *args, **kwargs):
         self.value = kwargs.pop('value', None)
-        super(Option, self).__init__(*args, **kwargs)
+        super(SetOption, self).__init__(*args, **kwargs)
 
 
-class HelpOption(_BaseOption):
-    """Option that prints the help message and exits."""
+class SetTrueOption(_BaseOption):
+    """Option that accepts no arguments, setting the destination value to True
+    if parsed and defaulting to False otherwise.
+    """
+
+    pass
```

Comment:
Question. Is pass supposed to be on this line according to flake8? I thought it was supposed to be right after the `"""`